pkg/providers/openstack/image.go

```diff
@@ -52,6 +52,81 @@ var (
 	ErrKeyType = errors.New("key type unsupported")
 )
 
+// imagePropertySchemaV2 defines what consitutes a valid image e.g. contains all the
+// required information to work correctly.  This is defined in:
+// https://github.com/unikorn-cloud/specifications/blob/main/specifications/providers/openstack/flavors_and_images.md.
+const imagePropertySchemaV2 = `{
```

Comment:
Might be slightly neater to put this in a file and use https://pkg.go.dev/embed. (You wouldn't have to cast to []byte later, too)